tests/mlflow/sparkml/test_sparkml_model_export.py

```diff
@@ -0,0 +1,54 @@
+import os
+import pandas as pd
+from sklearn import datasets
+
+import pyspark
+from pyspark.ml.classification import LogisticRegression
+from pyspark.ml.feature import VectorAssembler
+from pyspark.ml.pipeline import Pipeline
+from pyspark.version import __version__ as pyspark_version
+
+from mlflow import sparkml, pyfunc
+from mlflow.utils.environment import  _mlflow_conda_env
+
+from tests.helper_functions import score_model_in_sagemaker_docker_container
+
+def test_model_export(tmpdir):
+    conda_env = os.path.join(str(tmpdir), "conda_env.yml")
+    _mlflow_conda_env(conda_env, additional_pip_deps=["pyspark=={}".format(pyspark_version)])
+    iris = datasets.load_iris()
+    X = iris.data  # we only take the first two features.
+    y = iris.target
+    pandas_df = pd.DataFrame(X, columns=iris.feature_names)
+    pandas_df['label'] = pd.Series(y)
+    spark = pyspark.sql.SparkSession.builder \
+        .config(key="spark.python.worker.reuse", value=True) \
+        .master("local-cluster[2, 1, 1024]") \
+        .getOrCreate()
+    spark_df = spark.createDataFrame(pandas_df)
+    print("")
+    print(dir(tmpdir))
+    print(pandas_df)
+    print(spark_df.show())
```

Comment:
Probably should get rid of the prints by default in tests

Reply:
true, will do